game/screens/gameplay/view.py

```
@@ -33,15 +34,14 @@ def __init__(self, screen, key_labels):
 
         # Button layout — tight d-pad cross centered slightly above mid, space below
         cx, cy = W // 2, H // 2 - 40
-        s = 90    # arrow button size
-        gap = 100  # center-to-center distance
-
+        spah = 210; #height of space bar
```

Ruff (0.15.10)

[error] Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)